ui/packages/consul-ui/app/styles/components/empty-state/skin.scss

```diff
@@ -2,9 +2,6 @@
   color: $gray-500;
   background-color: $gray-010;
 }
-%empty-state > ul {
-  border-color: $gray-300;
-}
 %empty-state-header {
```

Comment:
Are we ok removing this line app wide? I had a look in the various figma files and it seems like sometimes this line is there and sometimes its not, all in all I'm not massively clear to me whether this needs removing app wide or only on certain empty states, @jnwright can you let us know whats required here?

Reply:
It can be removed 👍🏼